### DISCUSSION TITLE
How to deploy the Admin part to vercel.com?

### DISCUSSION THREAD

**Discussion by abarmawi** [payloadcms/payload#4558] [General]
I'm trying to host the Admin part only on https://vercel.com/
I was able to integrate the website with Vercel, but when accessing `/admin`, i get 404.

Couple you point me to a documentation for this?

**Comment by valschr** [comment]
You'll need to use https://github.com/payloadcms/next-payload for now. 

With payload moving to NextJS this package will be replaced in V3.0

**Reply by abd-resy** [reply]
Thanks @valschr , i will check this 👍
